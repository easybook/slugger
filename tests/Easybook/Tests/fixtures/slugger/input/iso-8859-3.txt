!"#$%&'()*+,-./
0123456789:;<=>?
@ABCDEFGHIJKLMNO
PQRSTUVWXYZ[\]^_
`abcdefghijklmno
pqrstuvwxyz{|}~
Ħ˘£¤Ĥ§¨İŞĞĴŻ
°ħ²³´µĥ·¸ışğĵ½ż
ÀÁÂÄĊĈÇÈÉÊËÌÍÎÏ
ÑÒÓÔĠÖ×ĜÙÚÛÜŬŜß
àáâäċĉçèééêëìíîï
ñòóôġö÷ĝùúûüŭŝ˙