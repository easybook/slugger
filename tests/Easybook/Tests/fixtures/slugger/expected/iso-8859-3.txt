
0123456789
abcdefghijklmno
pqrstuvwxyz
abcdefghijklmno
pqrstuvwxyz
hhisgjz
hhisgjz
aaaaccceeeeiiii
nooogoguuuuusss
aaaaccceeeeeiiii
nooogoguuuuus